plugins/colordialog/dialogs/colordialog.js

```diff
@@ -276,6 +276,22 @@ CKEDITOR.dialog.add( 'colordialog', function( editor ) {
 		title: lang.title,
 		minWidth: 360,
 		minHeight: 220,
+		onShow: function( evt ) {
+			if ( evt.data.colordialog && evt.data.colordialog.selectionColor ) {
```

Comment:
Reversing this conditional to create early return would allow us to flatten at least one indentation level:

```javascript
if ( !evt.data.colordialog || ! evt.data.colordialog.selectionColor ) {
	return;
}

// Rest of the code.
```